--- origsrc/rings-1.1/Makefile	2006-12-04 10:01:12.000000000 -0600
+++ src/rings-1.1/Makefile	2008-02-14 19:05:13.593750000 -0600
@@ -10,12 +10,11 @@
 OBJS= src/rings.o
 
 src/$(LIBNAME) : $(OBJS)
-	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) $(CFLAGS) $(LIB_OPTION) -o src/$(LIBNAME) $(OBJS)
+	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) $(CFLAGS) $(LIB_OPTION) -o src/$(LIBNAME) $(OBJS) $(LIBS)
 
 install:
 	mkdir -p $(LUA_LIBDIR)
 	cp src/$(LIBNAME) $(LUA_LIBDIR)
-	cd $(LUA_LIBDIR); ln -f -s $(LIBNAME) $T.so
 	mkdir -p $(LUA_DIR)
 	cp src/stable.lua $(LUA_DIR)
 
--- origsrc/rings-1.1/config	2006-12-15 12:04:16.000000000 -0600
+++ src/rings-1.1/config	2008-02-14 19:05:06.937500000 -0600
@@ -12,10 +12,10 @@
 LIB_OPTION= -shared #for Linux
 #LIB_OPTION= -bundle -undefined dynamic_lookup #for MacOS X
 
-LIBNAME= $T.so.$V
+LIBNAME= $T.dll
 
 # Compilation directives
-WARN= -O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -Wpointer-arith -pedantic
+WARN= -O2 -Wall -DPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -Wpointer-arith -pedantic
 INCS= -I$(LUA_INC)
 CFLAGS= $(WARN) $(INCS)
 CC= gcc
